hassio-to-pubsub-source/config.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Home Assistant to Pub/Sub",
+  "version": "0.0.3",
+  "slug": "hassio-to-pubsub",
+  "description": "Publishes Home Assistant events to Google Pub/Sub",
+  "arch": ["aarch64"],
+  "startup": "application",
+  "boot": "auto",
+  "homeassistant_api": true,
+  "options": {
+    "home_id": null,
+    "service_account_json": null,
+    "filter": {
+      "include_domains": [],
+      "exclude_domains": [],
+      "include_entity_globs": [],
+      "exclude_entity_globs": [],
+      "include_entities": [],
+      "exclude_entities": []
+    }
+  },
+  "schema": {
```

Comment:
We've made `gcp_project_id` a required option now, so it should be in the schema.